UtilityBeltTests/UtilityBeltDataTests/Extensions/XCTestCase+MeasureAndCatch.swift

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import Foundation
+import XCTest
+
+extension XCTestCase {
+    func measureAndCatch(file: StaticString = #file,
+                         line: UInt = #line,
+                         block: () throws -> Void) {
+        measure {
```

Comment:
self?

Reply:
I have the `self` rule enabled in `swiftformat`. I believe `measure` and other similar operations like `wait` and so on are not triggering or corrected because of how they are used within testing blocks.

I'm going to rely on `swiftlint` and `swiftformat` to handle items like this if applicable I think. Thoughts?

Reply:
I guess that works for me, but I would still add self when we catch them for consistency.

Reply:
I don't necessarily agree with this for `XCTestCase` methods, it's typical for test suites to have functions like this and, even for projects that focus on `self` in and outside of `Swift`, to exclude `self` for test-related blocks for clarity. Like I said, I think these are intentionally excluded as exceptions, _only_ within test suites. I'll dig a bit and see if there's a rule for it somewhere.

Ultimately, if our linter or formatter has a rule for something and doesn't trigger and can't have an option to do so, I'd lean on that being our standard. With that said, it also doesn't _remove_ `self`, so I wouldn't comment on it either way. If you think it's worth having as part of our Style Guide, can you open a PR and we can discuss as a team?